Common/src/main/java/com/tabnineCommon/lifecycle/PluginInstalled.kt

```diff
@@ -0,0 +1,41 @@
+package com.tabnineCommon.lifecycle
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.util.messages.MessageBusConnection
+import com.intellij.util.messages.Topic
+
+fun interface PluginInstalled {
+    companion object {
+        val PLUGIN_INSTALLED_TOPIC = Topic.create(
+            "Plugin Installed Notifier",
```

Comment:
since topics are registered globally across the process, having a unique name is essential:
```suggestion
            "com.tabnine.notifyPluginInstalled",
```
or something ^
(also why use spaces)

Reply:
I don't think the name matters and am not sure it is global

look at other topics in the code

Reply:
why take the chance tho